components/dropdown/dropdown-content-mixin.js

```diff
@@ -47,6 +51,14 @@ export const DropdownContentMixin = superclass => class extends RtlMixin(supercl
 				type: Number,
 				attribute: 'max-height'
 			},
+			/**
+			 * Override default mobile dropdown style. Specify one of 'tray' or 'dialog'.
+			 */
+			mobileTray: {
```

Comment:
Looks like we need to update the comment here -i.e. looks like these are not the accepted values.

Reply:
Is `bottom` going to be implemented as well?  I noticed some empty CSS selector further below.

Reply:
Yes, that's a TODO for now, The first story is just the left/right styles since they are very similar.